tests/JWT.Tests/DecodeTests.cs

```diff
@@ -1,14 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
+
+#if !NETCOREAPP1_0
 using System.Web.Script.Serialization;
+#endif
 
 using FluentAssertions;
 
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Xunit;
```

Comment:
Love this! Just wanted to make sure, was going to switch over Xunit anyway with v2